src/diffpy/pdfmorph/__init__.py

```diff
@@ -19,8 +19,11 @@
 # top-level import
 from diffpy.pdfmorph.pdfmorph_api import morph_default_config, pdfmorph, plot_morph
 
-# obtain version information
-from diffpy.pdfmorph.version import __version__
+top_level = [
+    pdfmorph,
+    plot_morph,
+    morph_default_config,
+]
```

Comment:
Cannot remove from `__init__.py`, added to top level. We can either add `# noqa` to the end of the import line or do this for top-level.